internal/helpers/helpers.go

```diff
@@ -219,6 +221,52 @@ func VariableExists(vars *[]LagoonEnvironmentVariable, name, value string) bool
 	return exists
 }
 
+// GetLagoonVariable returns a given environment variable
+func GetLagoonVariable(name string, scope []string, variables []LagoonEnvironmentVariable) (*LagoonEnvironmentVariable, error) {
+	for _, v := range variables {
+		scoped := true
+		if scope != nil {
+			scoped = false
```

Comment:
I think you could replace with all just with
scoped = constainsStr(scope, v.Scope)

Reply:
lines 229-232, I mean

Reply:
:see_no_evil: :force: